### DISCUSSION TITLE
DSEP Working Group - Call 2 - Feb 23rd, 2022

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/DSEP-Specification#17] [Announcements]
Attaching agenda of today's call

![Call 2 - Agenda](https://user-images.githubusercontent.com/52468749/155298744-aa9c3e97-4427-417d-9fb0-870380054b33.png)

**Comment by sujitrs** [comment]
[Presentation, Call Recording and Action Items for DSEP Working Group - Call 2 - Feb 23rd, 2022.pdf](https://github.com/beckn/DSEP-Specification/files/8129989/Presentation.Call.Recording.and.Action.Items.for.DSEP.Working.Group.-.Call.2.-.Feb.23rd.2022.pdf)

**Reply by bharatkashyap** [reply]
@sujitrs You can make these available as viewable PDFs by sharing a link to a file (instead of the file having to be downloaded). 
Also, the call recording links are not viewable unfortunately.
